src/transaction/utils.ts

```diff
@@ -412,7 +417,136 @@ function convertMemo(memo: Memo): { Memo: ApiMemo } {
   };
 }
 
+/**
+ * Autofills fields in a transaction.
+ *
+ * @param {Transaction} tx A transaction to autofill fields.
+ * @param {Client} client A client.
+ * @returns {Transaction} An autofilled transaction.
+ */
+async function autofillTransaction(
+  tx: Transaction,
```

Comment:
We'll want this on the client, so I think we'll want to define this as 

```ts
async function autofillTransaction(
    this: Client,
    tx: Transaction
```
and then 
```ts
class Client {
    autofillTransaction = autofillTransaction
}

Reply:
also agree that the `client` should be the first argument, as that makes usage with partial application (see above) or currying easier

Reply:
i also think this should just be called `autofill` since it's already in the `transaction/utils.ts` file - that makes it clear what is being autofilled